website/src/app/[lang]/[region]/(website)/(home)/(sections)/video.tsx

```diff
@@ -16,30 +16,28 @@ import {
 export async function Video({ lang }: { lang: LanguageCode }) {
 	const translator = await Translator.getInstance({
 		language: lang,
-		namespaces: ['website-home', 'common'],
+		namespaces: ['website-home', 'common', 'website-videos'],
 	});
 
 	return (
 		<BaseContainer className="flex flex-col justify-center py-8 md:pb-16">
 			<div className="overflow-hidden rounded-lg">
-				<VimeoVideo videoId={Number(translator.t('video.video-id'))} />
+				<VimeoVideo videoId={Number(translator.t('id.video-01'))} />
 			</div>
 			<div className="mt-2 self-end">
 				<Popover openDelay={0} closeDelay={200}>
 					<PopoverTrigger>
-						<Typography>{translator.t('video.credits-title')}</Typography>
+						<Typography>{translator.t('credits')}</Typography>
 					</PopoverTrigger>
 					<PopoverContent align="end" className="w-96">
 						<Table>
 							<TableBody>
-								{translator
-									.t<{ role: string; name: string }[]>('video.credits-content')
-									.map(({ role, name }, index) => (
-										<TableRow key={index}>
-											<TableCell className="p-1.5 font-medium">{role}</TableCell>
-											<TableCell className="p-1.5">{name}</TableCell>
-										</TableRow>
-									))}
+								{translator.t<{ role: string; name: string }[]>('video-01.credits').map(({ role, name }, index) => (
```

Comment:
Since merging the latest main into my campaign branch, I get the following error when calling `http://localhost:3001` (which redirects me to `http://localhost:3001/en/ch`):

<img width="1017" alt="image" src="https://github.com/socialincome-san/public/assets/1204548/59be7677-eb18-442b-a11e-6b21a2621a2f">

```
 src/app/[lang]/[region]/(website)/(home)/(sections)/video.tsx (38:76) @ map
 ⨯ TypeError: translator.t(...).map is not a function
    at Video (video.tsx:66:80)
digest: "791144539"
  36 |                                          <Table>
  37 |                                                  <TableBody>
> 38 |                                                          {translator.t<{ role: string; name: string }[]>('video-01.credits').map(({ role, name }, index) => (
     |                                                                                                                             ^
  39 |                                                                  <TableRow key={index}>
  40 |                                                                          <TableCell className="p-1.5 font-medium">{role}</TableCell>
  41 |                                                                          <TableCell className="p-1.5">{name}</TableCell>
```

Seems like `translator.t<{ role: string; name: string }[]>('video-01.credits')` resolves to `undefined`.

Reply:
@anthonyray had the same issue and mentioned it in a [Slack message](https://socialincome.slack.com/archives/C040MU2HHT8/p1707923501262169). 

The [proposed solution](https://socialincome.slack.com/archives/C040MU2HHT8/p1708076278056079):

> Erasing the website' `.next` folder and running `npm install` and `npm run website:serve `

might have fixed it – at least it got a 👍🏻 from @anthonyray (maybe it was also just to say thanks for the possible solution and it didn't do anything). Worth trying out.

Reply:
That indeed fixed it. Great stuff, thanks @ssandino